CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.58.7] - 2024-09-13
+Add WildcardResourceSubscriber for restli-resource-explorer subscribe to all resources
```

Comment:
Nit: might need a `-` at the start of this line. Also, I don't know if we need to mention `restli-resource-explorer` by name, no need to put the name of an internal app in the code + in theory any app may want to use `WildcardResourceSubscriber`